espnet/bin/asr_train.py

```diff
@@ -359,7 +269,7 @@ def main(cmd_args):
 
     # train
     logging.info('backend = ' + args.backend)
-    if args.num_spkrs == 1:
+    if not hasattr(args, 'num_spkrs'):
```

Comment:
Is this working?
We may need to change  https://github.com/espnet/espnet/blob/master/egs/wsj_mix/asr1/run.sh accordingly, right?
Can you do it? Or ask @simpleoier to do it.

Reply:
I reverted this part because asr_mix should not be included in this PR.